oci/pack.go

```diff
@@ -748,6 +748,10 @@ func (ocipack *ociPackage) Unpack(ctx context.Context, dir string) error {
 			initrdPath,
 			initrd.WithArchitecture(image.Architecture),
 			initrd.WithWorkdir(dir),
+
+			// We don't know the filesystem type of the initrd, so we set it to
+			// unknown. We let the initrd subsystem detect it.
+			initrd.WithType(initrd.FsTypeUnknown),
```

Comment:
This can be removed, it's not true that "unknown" is checked, it simply defaults to CPIO (which is correct), but this is not necessary.